java-vtl-model/src/main/java/no/ssb/vtl/model/VTLDate.java

```diff
@@ -57,6 +62,11 @@ public static VTLDate of(Instant instant) {
 
         return new VTLDate() {
             @Override
+            public Class<VTLDate> getType() {
+                return VTLDate.class;
```

Comment:
Why is getType() not implemented by VTLDate directly?

Reply:
It should! Good catch.